canonical-websites/snapcraft.io
# Use the Vanilla colour scheme for links and buttons

## Done

Use the Vanilla colour scheme for links and buttons

## QA

- Follow the instructions in README.md to get started
- Go to http://127.0.0.1:4000 and make sure links and buttons have the same styling as the latest vanilla framework 

## Issue / Card

[Issue 321 relates](https://github.com/canonical-websites/snapcraft.io/issues/321)
